lib/rules/selector-pseudo-class-case/index.js

```diff
@@ -61,6 +61,11 @@ const rule = function(expectation) {
             return;
           }
 
+          if (context.fix) {
+            rule.selector = rule.selector.replace(pseudo, expectedPseudo);
```

Comment:
You can also directly replace the value of `pseudoNode`, and receive it as the return value of `parseSelector`.